lib/modules/datasource/npm/get.spec.ts

```diff
@@ -146,7 +150,8 @@ describe('modules/datasource/npm/get', () => {
       .get('/renovate')
       .reply(200, { name: 'renovate' });
     setNpmrc(npmrc);
-    await getDependency(http, 'renovate');
+    const registryUrl = resolveRegistryUrl('renovate');
+    await getDependency(http, registryUrl, 'renovate');
```

Comment:
Shouldn't we add an expectation for the result?
The line below will be removed soon

Reply:
I guess so, but I was just keeping up the status quo